docs/helm/eks-deployment.md

```diff
@@ -8,13 +8,210 @@ manage, and scale containerized applications using Kubernetes on AWS. EKS runs
 availability zones to eliminate a single point of failure.
 
-The Enterprise configuration will deploy the following system:
+## Architectures
+
+### Enterprise diagrams
+
+The Enterprise configuration will deploy the following system (Alfresco
+Transform Service and Alfresco Search Enterprise are detailed in the next
+diagrams):
+
+```mermaid
+graph LR
+
+classDef alf fill:#0b0,color:#fff
+classDef aws fill:#fa0,color:#fff
+classDef k8s fill:#326ce5,stroke:#326ce5,stroke-width:2px,color:#fff
+classDef thrdP fill:#e098a6,color:#000
+
+Client("👥 Clients")
+
+subgraph Helm enterprise
+  direction LR
+  PersistentVolumeClaim_activemq-default-pvc(PersistentVolumeClaim: activemq-default-pvc):::k8s
+  PersistentVolumeClaim_data-acs-postgresql(PersistentVolumeClaim: data-acs-postgresql):::k8s
+  PersistentVolumeClaim_data-sync-postgresql(PersistentVolumeClaim: data-sync-postgresql):::k8s
+  PersistentVolumeClaim_repository-default-pvc(PersistentVolumeClaim: repository-default-pvc):::k8s
+
+  Deployment_activemq(Deployment: activemq):::thrdP
+  Deployment_alfresco-cc(Deployment: alfresco-cc):::alf
+  Deployment_alfresco-dw(Deployment: alfresco-dw):::alf
+  Deployment_alfresco-repository(Deployment: alfresco-repository):::alf
+  Deployment_alfresco-sync-service(Deployment: alfresco-sync-service):::alf
+  Deployment_share(Deployment: share):::alf
+
+  StatefulSet_elasticsearch-master(StatefulSet: elasticsearch-master):::thrdP
+  StatefulSet_postgresql-sync(StatefulSet: postgresql-sync):::thrdP
+  StatefulSet_postgresql-acs(StatefulSet: postgresql-acs):::thrdP
+
+  Ingress_alfresco-cc(Ingress: alfresco-cc):::k8s
+  Ingress_alfresco-dw(Ingress: alfresco-dw):::k8s
+  Ingress_alfresco-repository(Ingress: alfresco-repository):::k8s
+  Ingress_alfresco-sync-service(Ingress: alfresco-sync-service):::k8s
+  Ingress_share(Ingress: share):::k8s
+end
+
+  subgraph AWS
+    EFS:::aws
+    S3:::aws
+    EBS:::aws
+  end
+
+Client ---> Ingress_alfresco-cc --> Deployment_alfresco-cc
+Client ---> Ingress_alfresco-dw --> Deployment_alfresco-dw
+Client --> Ingress_alfresco-repository --> Deployment_alfresco-repository
+Client --> Ingress_share --> Deployment_share
+Client --> Ingress_alfresco-sync-service --> Deployment_alfresco-sync-service
+
+Deployment_share --> Deployment_alfresco-repository
+
+Deployment_alfresco-repository --> StatefulSet_postgresql-acs --> PersistentVolumeClaim_data-acs-postgresql --> EBS
+Deployment_alfresco-repository --> Deployment_activemq
+Deployment_alfresco-repository --> StatefulSet_elasticsearch-master
+
+Deployment_alfresco-sync-service --> StatefulSet_postgresql-sync --> PersistentVolumeClaim_data-sync-postgresql --> EBS
+Deployment_alfresco-sync-service --> Deployment_activemq
+Deployment_alfresco-sync-service --> Deployment_alfresco-repository
+
+Deployment_alfresco-repository --> PersistentVolumeClaim_repository-default-pvc --> EFS
+PersistentVolumeClaim_repository-default-pvc -.-> EBS
```

Comment:
What is this dashed line meaning? optionnal?
I think we could simply remove non-optimal (optional options from the graph)

Reply:
the idea was to make it look like an alternative, but if needs to be explained maybe not worth it, let me drop it